src/Metadata/PropertyMetadata.php

```diff
@@ -33,6 +33,16 @@ class PropertyMetadata extends BasePropertyMetadata
      */
     public $serializedName;
 
+    /**
+     * @var string|null
+     */
+    public $unionDiscriminatorField;
```

Comment:
When we moved the code to the type, those fields seems to be not needed anymore. :)

Reply:
hmm, can you please clarify? im not sure I understand